about.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;500;700&family=Lato:wght@400;700;900&family=Poppins:ital,wght@0,200;0,400;0,600;1,500&display=swap" rel="stylesheet">
+<link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+<link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+    <link rel="stylesheet" href="about.css">
+  <title>Concert Page</title>
+</head>
+<body>
+  <section class="introduction">
+    <div id="overlay">
+      <div id="close-container" onclick="menuClose()">
+        <a id="closer" onclick="menuClose()">
+          <img src="./images/Icon.png" alt="Close" onclick="menuClose()">
+        </a>
+      </div>
+      <div id="menucontainer">
+          <a href="" onclick="menuClose()">Home</a>
+          <a href="" onclick="menuClose()">About</a>
+          <a href="" onclick="menuClose()">Program</a>
+          <a href="" onclick="menuClose()">Join</a>
+          <a href="" onclick="menuClose()">News</a>
+      </div>
```

Comment:
- [ ] Mobile menu links are not functioning as they are supposed to. Kindly fix the issue for both the home page and about page 👍 

![Animation2](https://user-images.githubusercontent.com/50721479/190718208-ee471898-be1f-4498-9a38-b93ece4916f4.gif)